Code/LauncherUnified/Platform/Windows/Launcher_Windows.cpp

```diff
@@ -10,6 +10,25 @@
 #include <AzCore/Math/Vector2.h>
 #include <AzCore/Memory/SystemAllocator.h>
 
+#if LY_HEADLESS
+int main(int argc, char* argv[])
+{
+    const AZ::Debug::Trace tracer;
+    InitRootDir();
+
+    using namespace O3DELauncher;
+
+    PlatformMainInfo mainInfo;
+
+    mainInfo.m_instance = GetModuleHandle(0);
+
+    mainInfo.CopyCommandLine(argc, argv);
+
+    ReturnCode status = Run(mainInfo);
+
+    return static_cast<int>(status);
+}
+#else
```

Comment:
Realisticially, the entire function doesn't need to be `#if`, only the signature of main vs WinMain and the call to `CopyCommandLine` needs to be protected.